packages/reactivity/src/baseHandlers.ts

```diff
@@ -63,9 +63,10 @@ function createGetter(isReadonly = false, shallow = false) {
 
     const res = Reflect.get(target, key, receiver)
 
+    const keyIsSymbol = isSymbol(key)
     if (
-      isSymbol(key)
-        ? builtInSymbols.has(key)
+      keyIsSymbol
+        ? builtInSymbols.has(key as symbol)
```

Comment:
reusing the result of `isSymbol(key)` will disable type guard, so have to add `as` manually 😣